R/descriptives.R

```diff
@@ -1949,11 +1855,9 @@ DescriptivesInternal <- function(jaspResults, dataset, options) {
         errorMessage <- .descriptivesCheckPlotErrors(dataset, c(v1, v2, split), obsAmount = "< 2")
         if (is.null(errorMessage)) {
 
-          scatterData <- apply(dataset[, c(v1, v2)], 2, as.numeric) # ensure nominal ints are numeric
-
           p <- try(jaspGraphs::JASPScatterPlot(
-            x                 = scatterData[, v1],
-            y                 = scatterData[, v2],
+            x                 = dataset[, v1],
+            y                 = dataset[, v2],
```

Comment:
```suggestion
            x                 = dataset[[v1]],
            y                 = dataset[[v2]],
```
If I'm assuming correctly that `v1` and `v2` are length 1 variables.